configure

```diff
@@ -154,9 +154,12 @@ my @specs = # Array of array-refs, each inner array is an option group which
     'java:s', 'Java development kit (use JAVA_HOME)',
     'jboss:s', 'JBoss application server (use JBOSS_HOME)',
     'ant:s', 'Ant for JBoss (use ANT_HOME or system pkg)',
-    'wireshark:s', 'Wireshark (use WIRESHARK_SRC)',
-    'glib:s', 'GLib for wireshark (use GLIB_ROOT or system pkg)',
-    'rapidjson', 'RapidJSON for wireshark itl (optional)',
+    'wireshark:s', 'Wireshark 1.x Source (use WIRESHARK_SRC)',
+	'wireshark2:s', 'Wireshark 2.x Source (use WIRESHARK_SRC)',
```

Comment:
Replace tabs with spaces on the 3 lines starting here

Reply:
I thought I got all those but I think I was just greping `tools/dissector`.